### PR TITLE
Add check-manifest as pre-commit-hook

### LINKED ISSUE #100: check-manifest as pre-commit-hook
Hi,

thanks for the package. MANIFEST.in was always a mystery for me, but your package makes it better.

Could you make your package available as a pre-commit hook? It is simple as adding a ``.pre-commit-hooks.yaml`` to your package. The content is detailed here: https://pre-commit.com/#new-hooks.

The advantage is that people can easily integrate your package in their workflow and check before each commit whether the MANIFEST.in is sufficiently specified.

Best

### PR DESCRIPTION
**Pull request by tobiasraabe**
This PR resolves #100 and adds ``.pre-commit-hooks.yaml`` and a small section to the readme. This section is totally optional and can be deleted if deemed unnecessary.

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-   id: check-manifest
```
Comment (mgedmin) [inline .pre-commit-hooks.yaml]: My YAML linter complains

> hyphens: too many spaces after hyphen

(I can fix that after merging if you don't want to.)

File: README.rst [hunk 2]
```diff
@@ -125,6 +125,21 @@ ignore-bad-ideas
...
+---------------------------
+
+With `pre-commit <https://pre-commit.com>`_, check-manifest can be part of your
+git-workflow. Add the following to your ``.pre-commit-config.yaml``.
+
+.. code-block:: yaml
+
+    repos:
+    -   repo: https://github.com/mgedmin/check-manifest
+        rev: "0.38"
```
Comment (mgedmin) [inline README.rst]: Presumably this should be 0.39, because there's no .pre-commit-hooks.yaml in https://github.com/mgedmin/check-manifest/tree/0.38/ ?

**Comment by mgedmin** [comment]
Oops, forgot to check Travis status.  It fails because

```
$ python check_manifest.py
lists of files in version control and sdist do not match!
missing from sdist:
  .pre-commit-hooks.yaml
suggested MANIFEST.in rules:
  include *.yaml
The command "python check_manifest.py" exited with 1.
```

Would you mind updating MANIFEST.in this way?

File: README.rst (hunk 2 above)
Comment (tobiasraabe) [reply README.rst]: Fixed.

File: .pre-commit-hooks.yaml (hunk 1 above)
Comment (tobiasraabe) [reply .pre-commit-hooks.yaml]: I know and it was weird for me, too. I think this is just how the author formats the yaml. See here the following two examples: https://pre-commit.com/#docker_image. If you want a different format, it is fine.

**Comment by mgedmin** [comment]
Thank you!

**Comment by mgedmin** [comment]
check-manifest 0.39 is out on PyPI!

**Pull request merged**